### PR TITLE
Update maintaining.rst

### PR DESCRIPTION
**Pull request by Andersson007**
Update maintaining.rst

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

**Comment by Andersson007** [comment]
ready for review

File: maintaining.rst [hunk 1]
```diff
@@ -255,6 +259,8 @@ In addition to the paragraph above, here is a checklist:
   * Put information how people can complain in your ``README`` and ``CONTRIBUTING`` file.
   * Links to the `contributing.rst <contributing.rst>`_ and `Quick-start guide <create_pr_quick_start_guide.rst>`, and other documentation in ``README``.
   * Add and keep updated the ``CONTRIBUTORS`` and ``MAINTAINERS`` files.
+  * If the collection is a part of Ansible, mention it in README.
```
Comment (felixfontein) [inline maintaining.rst:262]:
```suggestion
  * If the collection is a part of Ansible, mention it in ``README``.
```

Comment (Andersson007) [reply maintaining.rst]: committed, thanks!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Pull request merged**